src/subscribe-substrate/lib/postgres.ts

```diff
@@ -595,11 +601,60 @@ export const deleteCommentActivityWithActivityStream = async (userId: string,com
         LEFT JOIN df.account_followers ON df.activities.account = df.account_followers.following_comment_id WHERE comment_id = $2)
     RETURNING *`
   const hexCommentId = encodeStructId(commentId);
-  const params = [userId, hexCommentId];
+  const params = [ userId, hexCommentId ];
   try {
     const res = await pool.query(query, params)
     console.log(res.rows)
   } catch (err) {
     console.log(err.stack);
   }
 }
+
+export const updateUnreadNotifications = async (account: string, activityId: number) => {
+  const defaultUnreadCount = 1;
```

Comment:
Why `1`, not `0`?

Reply:
We use this number only if we adding new line to notification counter. This happens when we should update counter but there is no row with needed account.